te_markdown/domainexperts.md

```diff
@@ -1,18 +1,33 @@
 ---
 layout: page
-title: Domain Experts
+title: Test Your Domain
 permalink: /te_markdown/domainexperts/
 ---
 
- Write your first test case in 5 minutes (link)
-  What are test cases? (link)
-  
-  this text is on the next page
-  - Tests or test cases are executable test descriptions that implement a test specification
-  - They are written in a language that is largely designed by the tester
-     
-        * Create a hero named "Sancho"
-        Component: MyHeroes
-        - Enter "Sancho" into <Name>
-        - Press <Add>
-     
+User acceptance testing is a collaborative activity involving domain experts, testers, and developers. Therefore, effective and efficient tools have to address the different needs of each of these roles. In particular, they must not expose technical aspects that would shut out less tech-savvy users that nevertheless need to participate in the process. 
+For example, tools embedded into heavy-weight integrated development environments expose far too many unneeded features and capabilities that distract from the task at hand, and unnecessarily steepen the learning curve. The same is true for using existing general-purpose programming languages to express test specifications and test cases, that were not build with these use cases in mind.
+
+**Test-Editor-Web** is designed specifically so that domain experts are empowered to write testable specifications, without requiring deep technical knowledge.
+
+To this end, the languages used to express test specifications and test cases are tailored specifically towards their respective use cases. They have been purpose-built from scratch, as opposed to extending an existing programming language, so that no unneeded and potentially confusing confusing concepts are exposed, allowing to express test specification and test cases in a clear and concise manner. 
+
+At the same time, the languages are *customizable*, so that concepts from the application domain and the software under test can be defined and given semantics to be used directly in tests, making them natural to read and understand for domain experts.
```

Comment:
```suggestion
At the same time, the languages are *customizable*, so that concepts from the application domain and the software under test can be defined and given semantics to be used directly in tests, making them natural to read and understandable for domain experts.
```

Reply:
Same comment as above